utils/build-validate-quickstart-artifact.ts

```diff
@@ -0,0 +1,121 @@
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+
+import Quickstart from "./lib/Quickstart";
+import DataSource from "./lib/DataSource";
+import Alert from "./lib/Alert";
+import Dashboard, { DashboardConfig } from "./lib/Dashboard";
+import Ajv, { type ErrorObject } from 'ajv';
+import { QuickstartConfig, QuickstartConfigAlert } from './types/QuickstartConfig';
+import { DataSourceConfig } from './types/DataSourceConfig';
+
+type ArtifactSchema = any;
```

Comment:
Do we want to type this out? I think id be fine either way since its just defining the type of the schema.

Reply:
I was just gonna leave it as `any` for now and we can determine if we wanna type it further depending on whether we need to use the type anywhere else. I didn't see much harm in just leaving this definition in for now though.